p02_8.py

```diff
@@ -0,0 +1,6 @@
+def ptoq( adX ):
+
+	adY = sorted( adX )
+	for i in range( len( adY ) ):
+		adY[i] = adY[i] * len( adY ) / ( i + 1.0 )
+	return adY
```

Comment:
In general, nice work. 
Please write a `test.py` file and test all functions you wrote with the examples provided in the prompts. 
This is called [test-driven development](https://zh.wikipedia.org/wiki/%E6%B5%8B%E8%AF%95%E9%A9%B1%E5%8A%A8%E5%BC%80%E5%8F%91), which is a very useful principle